src/vtools-analysis.py

```diff
@@ -90,9 +90,10 @@ def run_opencv_analysis(infile, add_mse, debug):
                 if diff_msey is None
                 else (math.log10(diff_msey) if diff_msey != 0.0 else "-inf")
             )
+
             psnr_y = (
                 None
-                if log10_msey is None
+                if ((log10_msey is None) | (log10_msey == "-inf"))
```

Comment:
you want to use "((log10_msey is None) or (log10_msey == "-inf"))" here, right? ("or" instead of "|")

Reply:
Sure. Less work.